sdk/src/metrics/state/temporal_metric_storage.cc

```diff
@@ -112,7 +112,7 @@ bool TemporalMetricStorage::buildMetrics(CollectorHandle *collector,
   else
   {
     merged_metrics->GetAllEnteries(
-        [](const MetricAttributes &attr, Aggregation &aggr) { return true; });
+        [](const MetricAttributes & /* attr */, Aggregation & /* aggr */) { return true; });
```

Comment:
This one also can be removed - line 114, 115

Reply:
Done